easybuild/easyconfigs/s/Subversion/Subversion-1.14.2-GCCcore-11.3.0.eb

```
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'Subversion'
+version = '1.14.2'
+
+homepage = 'https://subversion.apache.org/'
+description = " Subversion is an open source version control system."
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = [
+    'https://apache.belnet.be/%(namelower)s',
+    'http://www.eu.apache.org/dist/%(namelower)s',
+    'http://www.us.apache.org/dist/%(namelower)s',
+    'https://archive.apache.org/dist/%(namelower)s',
+]
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['Subversion-1.12.0-no_swig.patch']
+checksums = [
+    'c9130e8d0b75728a66f0e7038fc77052e671830d785b5616aad53b4810d3cc28',  # subversion-1.14.2.tar.bz2
+    '539ea2118f958d152d78438c81649eb727ff0b2e8491295702ee98e1f922041f',  # Subversion-1.12.0-no_swig.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('Autotools', '20220317'),
+]
+
+dependencies = [
+    ('APR', '1.7.0'),
+    ('APR-util', '1.6.1'),
+    ('SQLite', '3.38.3'),
+    ('zlib', '1.2.12'),
+    ('lz4', '1.9.3'),
+    ('utf8proc', '2.8.0'),
```

```suggestion
    ('utf8proc', '2.7.0'),
```

We have https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/u/utf8proc/utf8proc-2.7.0-GCCcore-11.3.0.eb

Or is there a need to use a newer version?